srcpkgs/fwupd/template

```diff
@@ -83,6 +71,6 @@ fwupd-devel_package() {
 		vmove "usr/lib/*.so"
 		vmove usr/share/gir-1.0
 		vmove usr/share/vala
-		vmove usr/share/gtk-doc
+#		vmove usr/share/gtk-doc
```

Comment:
what's going on here? it installs docs to /usr/share/doc

Reply:
see above

Reply:
removing this line means documentation is installed to the main package, we don't want that

Reply:
Sorry. What should it be then, `vmove usr/share/doc` ?

Reply:
it seems to work, let me know in case you're unhappy with that.

Reply:
yes, the documentation moved from /usr/share/gtk-doc/ to /usr/share/doc/